src/Jmikola/AutoLogin/Http/Firewall/AutoLoginListener.php

```diff
@@ -34,11 +35,11 @@ class AutoLoginListener implements ListenerInterface
      * @param AuthenticationManagerInterface                 $authenticationManager
      * @param string                                         $providerKey
      * @param string                                         $tokenParam
-     * @param LoggerInterface                                $logger
+     * @param PsrLoggerInterface|HttpKernelLoggerInterface   $logger
      * @param EventDispatcherInterface                       $dispatcher
      * @param array                                          $options
      */
-    public function __construct($securityContext, AuthenticationManagerInterface $authenticationManager, $providerKey, $tokenParam, LoggerInterface $logger = null, EventDispatcherInterface $dispatcher = null, array $options = array())
+    public function __construct($securityContext, AuthenticationManagerInterface $authenticationManager, $providerKey, $tokenParam, $logger = null, EventDispatcherInterface $dispatcher = null, array $options = array())
```

Comment:
@jmikola 

Out of scope of this PR, but how about starting to wrap the constructor arguments here?